source/adminguide/accounts.rst

```diff
@@ -366,15 +366,15 @@ the user are used.
        #. CloudStack searches for it in LDAP by the configured
           ``ldap.username.attribute``.
 
```

Comment:
Observed the changes in this file, in other PR: https://github.com/apache/cloudstack-documentation/pull/223